map.css

```diff
@@ -7,3 +7,79 @@
   height: 40vw;
   width: 100%;
 }
+
+#searchMap {
```

Comment:
!! we fixed that yesterday, didn't we? styling by IDs is highly ineffective and not maintainable. please research about BEM or Maintainablecss.com and follow the class-naming rules :)